pkg/operator/staticpodoperator/controlloops/deployment_controller.go

```diff
@@ -0,0 +1,257 @@
+package controlloops
```

Comment:
the package name is a little bit funky ;-)

Reply:
> the package name is a little bit funky ;-)

what package would you like?